libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.ts

```diff
@@ -60,6 +62,13 @@ export class SkyAgGridCellEditorLookupComponent
 
     this.#triggerType = SkyAgGridCellEditorUtils.getEditorInitialAction(params);
     const control = this.editorForm.get('selection');
+    this.ariaLabel = params.skyComponentProperties?.ariaLabelledBy
+      ? undefined
+      : params.skyComponentProperties?.ariaLabel ||
+        params.colDef.headerName ||
+        params.colDef.headerTooltip ||
+        params.colDef.field ||
+        params.colDef.colId;
```

Comment:
Can any of those properties change during a change detection cycle? Do we need to respond to those changes?

Reply:
There is not an API to change a column definition. There is an event for "new columns loaded" but that doesn't make sense. Renderers have a `refresh` method but editors do not.